src/components/cylc/Info.vue

```diff
@@ -84,6 +84,42 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         </v-expansion-panel-text>
       </v-expansion-panel>
 
+      <v-expansion-panel class="run-mode-panel">
+        <v-expansion-panel-title color="blue-grey-lighten-2">
+          Run Mode
+        </v-expansion-panel-title>
+        <v-expansion-panel-text>
+          <v-icon>{{ runModeIcon }}</v-icon>  {{ runMode }}
+        </v-expansion-panel-text>
+      </v-expansion-panel>
+
+      <v-expansion-panel class="xtriggers-panel">
+        <v-expansion-panel-title color="blue-grey-lighten-2">
```

Comment:
I was going for striping with these panels because the default style doesn't make it overly obvious that this is a collection of panels when there are lots of them and they are all collapsed. I'm sure we'll come up with something nicer in the future.

```suggestion
        <v-expansion-panel-title color="blue-grey-lighten-1">
```

Reply:
I think this got wiped out in a force-push or something?